lib/rubocop/cop/rails/match_route.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop identifies places where defining routes with `match`
+      # can be replaced with a specific HTTP helper method.
```

Comment:
I think this can be "HTTP method", no need for `helper`, since that may cause confusion with things in `app/helpers`.